src/Gren/Kernel/Array.js

```diff
@@ -145,3 +145,16 @@ var _Array_sortWith = F2(function (fn, array) {
     return ord === __Basics_EQ ? 0 : ord === __Basics_LT ? -1 : 1;
   });
 });
+
+var _Array_range = F2(function(from, to) {
+  if (from > to) { return []; }
+  if (from === to) { return [from]; }
+
+  var result = [];
+
+  for (var i = from; i <= to; i++) {
+      result.push(i);
+  }
+
+  return result;
```

Comment:
I think this can be implemented in pure Gren by relying on `initialize`.

I hope to avoid writing kernel code whenever possible, as it carries a rather stiff maintainance cost (hard-to-diagnose errors, no typechecking, etc.)

Reply:
@robinheghan I've implemented `range` in gren using `initialize` (https://github.com/gren-lang/core/pull/37/commits/8b0a1a711c417cc27bd3ca19494cb1736695633e) and I've added myself to the contributors (https://github.com/gren-lang/core/pull/37/commits/d2afe33a24d12fe25bcdcabb945cc98ebfe4221e). Don't hesitate if there is anything you want me to change :wink: